packages/react-reconciler/src/ReactFiberHooks.js

```diff
@@ -1355,6 +1358,8 @@ export const ContextOnlyDispatcher: Dispatcher = {
   useResponder: throwInvalidHookError,
   useDeferredValue: throwInvalidHookError,
   useTransition: throwInvalidHookError,
+
+  unstable_avoidThisRender,
```

Comment:
This is wrong. Should be a method that throws an error. ("unstable_avoidThisRender() should not be called outside render...")